contents/data/nosql/NoSQL.md

```diff
@@ -0,0 +1,437 @@
+<frontmatter>
+  title: Introduction to NoSQL
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to NoSQL
+
+**Author(s): [Ang Ze Yu](https://github.com/ang-zeyu)**
+
+**Reviewer(s): [Neil Brian](https://github.com/nbriannl), [James Pang](https://github.com/jamessspanggg), [Daryl Tan](https://github.com/openorclose), [Yash Chowdhary](https://github.com/yash-chowdhary)**
+
+<box type="info">
+Basic knowledge of SQL databases is assumed. If not, give the <a href="../sql/SQL.html">SQL article</a> a read first!
+</box>
+
+<box id="article-toc" style="background-color: #dee2e62b">
+
+##### Table of contents { .no-index .border-bottom .pb-1 }
+
+* [What is NoSQL‎](#what-is-nosql)
+* [Key Characteristics of NoSQL Databases](#key-characteristics-of-nosql-databases)
+  * [Powerful and Simple CRUD Operations](#1-powerful-and-simple-crud-operations)
+  * [Schema-Less Data](#2-schema-less-data)
+  * [Straightforward Expression of Relations in Data](#3-straightforward-expression-of-relations-in-data)
+* [Why NoSQL](#why-nosql)
+  * [1. Highly Suited for Iterated Development](#1-highly-suited-for-iterated-development)
+  * [2. Easy Horizontal Scaling](#2-easy-horizontal-scaling)
+  * [3. Widespread Adoption](#3-widespread-adoption)
+* [Caveats of NoSQL‎](#caveats-of-nosql)
+  * [Lack of Standardisation](#1-lack-of-standardisation)
+  * [Not Suited for Complex Relational Queries‎](#2-not-suited-for-complex-relational-queries)  
+* [How to get Started With NoSQL?‎](#how-to-get-started-with-nosql)
+
+{.px-3}
+
+</box>
+
+---
+
+## What is NoSQL?
+
+**Non-Structured Query Language** (**NoSQL**) is a wide set of implementations of query technologies used to retrieve and store data in a non-tabular format.
```

Comment:
It's good if you can also mention other names such as 'not only SQL', at least as a tooltip.